charts/kong/values.yaml

```diff
@@ -513,14 +513,14 @@ dblessConfig:
 ingressController:
   enabled: true
   image:
-    repository: kong/kubernetes-ingress-controller
-    tag: "2.10"
+    repository: kong/nightly-ingress-controller
+    tag: "2023-07-18"
     # Optionally set a semantic version for version-gated features. This can normally
     # be left unset. You only need to set this if your tag is not a semver string,
     # such as when you are using a "next" tag. Set this to the effective semantic
     # version of your tag: for example if using a "next" image for an unreleased 3.1.0
     # version, set this to "3.1.0".
-    effectiveSemver:
+    effectiveSemver: 2.11.0
```

Comment:
If we discard these changes we could merge it already, even before KIC 2.11 release, right?

Reply:
Yeah, that was why I'd marked 58024d5888bc7c5c4595d918e588057fb86ba700 `tmp` and said:

> This needs the nightly KIC commit removed before merging. It's in place for now to demonstrate behavior on that version. test1 still uses 2.10 to validate the backwards-compatible path.

IDK a great way in GitHub to handle stuff that needs future changes elsewhere to properly demonstrate/test for review without some manual commit juggling. Have removed it and the hold label. https://github.com/Kong/charts/actions/runs/5684355422 is the latest run that checked both new and old with that temp commit.